### PR TITLE
Fix 6679

### PR DESCRIPTION
**Pull request by ZerooCool**
https://github.com/go-gitea/gitea/issues/6679

Please check the following:

1. Make sure you are targeting the `master` branch, pull requests on release branches are only allowed for bug fixes.
2. Read contributing guidelines: https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md
3. Describe what your pull request does and which issue you're targeting (if any)

**You MUST delete the content above including this line before posting, otherwise your pull request will be invalid.**

### CODE REVIEW

File: public/less/_repository.less [hunk 1]
```diff
@@ -1075,6 +1075,7 @@
             }
             pre {
                 margin: 0;
+                white-space: pre-wrap;
```
Comment (kolaente) [inline public/less/_repository.less]: That's changing the code-diff view, not the readme.

**Review by kolaente — CHANGES_REQUESTED** [review]

**Comment by ZerooCool** [comment]
Ok, that's not what's wanted then.
Here is the view of the HTML code (WordPress README displayed by default on our project) that I want to change because the code overflows.

An idea of CSS to modify?
In my online tests, the following CSS I modify: 
/vendor/plugins/semantic/semantic.min.css

**Comment by saitho** [comment]
Adding the CSS property to \<pre> tags inside the `.non-diff-file-content .plain-text` class should be fine.
=> on file **public/less/_repository.less** add the following after line 327:
```
pre {
  white-space: pre-wrap;
}
```

Make sure to indent the lines properly. Rebuild the CSS file with `make generate-stylesheets` and commit that as well (as suggested by the DroneCI error message).

**Comment by lafriks** [comment]
`make generate-stylesheets`

**Comment by ZerooCool** [comment]
This pull request was not good.
Please, read the last message her : https://github.com/go-gitea/gitea/issues/6679

Please, apply the good request, i can't !

**Comment by zeripath** [comment]
I'm going to close this as a duplicate

**Comment by ZerooCool** [comment]
Yes thank you.

**Pull request closed without merging**